### PR TITLE
Project chatbot

### PR DESCRIPTION
**Pull request by CeciliaMichele**
By Fanny Cardell & Cecilia Rådén

### CODE REVIEW

**Comment by malmen237** [comment]
You're missing the link to the deployed site, please add it.

**Comment by CeciliaMichele** [comment]
Link to Netlify-website: https://fixmechatbot.netlify.app/

File: code/script.js [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
 
-
 // Declare your functions after this comment
+const botReply = (message) => {
+  showMessage(message, 'bot');
+}
+
+const userReply = (message) => {
+  showMessage(message, 'user');
+}
```
Comment (marwebdesign) [inline code/script.js:18]: A nice way to simplify the code!

File: code/script.js [hunk 2]
```diff
@@ -27,25 +38,82 @@ const showMessage = (message, sender) => {
...
       </section>
-    `
+    `;
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
 }
 
-// Starts here
+// Question 1: bot asks for name.
```
Comment (marwebdesign) [inline code/script.js:47]: Good with clarfication to know what the code does!

File: code/script.js [hunk 3]
```diff
@@ -27,25 +38,82 @@ const showMessage = (message, sender) => {
...
+});
+chatSection.onsubmit = ()=> {
+}
+
 // When website loaded, chatbot asks first question.
 // normally we would invoke a function like this:
 // greeting()
 // But if we want to add a little delay to it, we can wrap it in a setTimeout:
 // setTimeout(functionName, timeToWaitInMilliSeconds)
 // This means the greeting function will be called one second after the website is loaded.
```
Comment (marwebdesign) [inline code/script.js:117]: The code looks cleaner if these types of comments were removed before published.

File: code/index.html [hunk 4]
```diff
@@ -8,24 +8,39 @@
...
+
+<!--- <p>testing</p>
+-->
+
+      </section>
 
       <div class="input-wrapper" id="input-wrapper">
         <form id="name-form">
-          <label for="name-input">name</label>
+          <label for="name-input">Whats your name?</label>
```
Comment (marwebdesign) [inline code/index.html:30]: Can this be removed? Is it needed? The bot asks the same question.

**Review by marwebdesign — CHANGES_REQUESTED** [review]
Overall a functioning webpage! Well done! 

We are missing a step from Iteration 5, where there were supposed to be some conditionals. Try to add som if else-statements somewhere. If it helps, you can use something similar to ours that we added as a reply if the user doesn't type anything in the input wrapper when supposed to write their name. 

     if (reply !== '') {
        setTimeout (() => showOptions(reply), 1000);
      } else {
        setTimeout (() => botReply('That is not a valid answer'), 300); 
      }


The code would look nicer and be easier to read if it was cleaned up a bit. Some empty lines could be removed, and the indentation could be looked over. 

You did very good with the naming of variables! We also liked the using of emojis to make your buttons more varied (: 

GOOD JOB! 

/Maria Westling and Linda Malm

**Pull request closed without merging**